Tools/Create-Azure-Sentinel-Solution/V3/CCP_README.md

```diff
@@ -0,0 +1,329 @@
+1. Data Connector Definition - Consider this an a UI page shown after deployment to users and contains instruction steps and connectorUiConfig section.
+2. Data Connectors (Poller) - This will contain details to connect to your REST API and has a poller details.
+3. Data Collection Rule - Specify, what data should be collected, how to transform that data, and where to send it.
+4. Tables (Optional) - Stores your data logs.
+
+## How to Use ?
+1. In Data input file under Azure-Sentinel/Solutions, open the solution you want to add a CCP connector.
+2. Open data input file and under "Data Connectors" array object add a path to new dataConnectorDefinition file.
+3. File name to dataConnectorDefinition should be meaningful and recommended to have a suffix "Definition" attached to the name at the end e.g. dataConnectorDefinition.json.
+4. Navigate to your "Solutions/*yourSolutionName*/Data Connectors" folder and create a new folder for ccp suffix at the end e.g: DataConnectorDefinitionName_ccp".
+5. Once a folder for CCP is created then add/create 4 files dataConnectorDefinition.json, dataPoller.json, DCR.json and table.json. Here table.json file is optional. Also, you can specific a different naming to the files but having a suffix of "Definition", "Poller", "DCR", "Table" in the file name at the end will be easy to correlate. Below is an example folder file structure for CCP.
+
+    ![Alt text](ccp-folder-structure.png)
+
+6. There can be only 1 data definition file per CCP connector. If you have multiple CCP connectors then create multiple data connector definition files and specify the file paths in the data folder input file under "Data Connectors" array. There is no need to specify other file paths for poller, dcr or tables in data input file. 
+7. dataConnectorDefinition file is the starting point for any CCP connector and this file path should only be specified in data folder input file under "Data Connectors" array.
+
+    ![Alt text](dataInputDataConnectorsArray.png)
+
+8. Mapping between dataConnectorDefinition and Poller file should present. Poller file mapping with DCR should be present. If mapping is not present and if present but mapping values are not correct then packaging will fail.
+
+    #### *<span style="color:blue">Note:</span> <span style="color:green">Each of this files depend on mapping with other files which is specified in "How to Use ?" section of this document. Eg: Data definition file should have a mapping with Data poller file. Data poller file should have a mapping with DCR file. DCR file should have a mapping with Tables file. Tables file is optional and so, if there is no mapping from DCR with table then remove the "outputStream" property from "DCR" file. You can skip removing property "outputStream" from DCR file, if it is a Standard table eg: "outputStream"="Microsoft-ASimNetworkSessionLogs" in DCR is a Standard table and there is no need to add table file.</span>*
```

Comment:
Again, very confusing to read this. Can we have simple relation png file on how it is mapped to easily understand?

Reply:
updated